buildshiprun/handler.go

```diff
@@ -162,15 +162,15 @@ func Handle(req []byte) string {
 			Image:   imageName,
 			Network: "func_functions",
 			Labels: map[string]string{
-				"Git-Cloud":      "1",
-				"Git-Owner":      event.Owner,
-				"Git-Repo":       event.Repository,
-				"Git-DeployTime": strconv.FormatInt(time.Now().Unix(), 10), //Unix Epoch string
-				"Git-SHA":        event.SHA,
-				"faas_function":  serviceValue,
-				"app":            serviceValue,
-				"com.openfaas.scale.min": scalingMinLimit,
-				"com.openfaas.scale.max": scalingMaxLimit,
+				"com.openfaas.cloud.git-cloud":      "1",
```

Comment:
Suggest we put this prefix into a const and store it in the SDK package?

Reply:
It's a good idea. Should I put only `com.openfaas.cloud` as single const or the whole `com.openfaas.cloud.git-cloud` and add consts for all the variables like `const gitRepo = `com.openfaas.cloud.git-repo`

Reply:
@martindekov cc @bartsmykla please raise a quick PR to the SDK component to add a const for this prefix. I'll then do a merge / release and you can vendor it here.

Reply:
Ok I've already done that in the unpushed changes But will make it into another PR